.github/workflows/library.yml

```diff
@@ -105,6 +105,15 @@ jobs:
 
     runs-on: ${{ matrix.os }}-latest
 
+    services:
```

Comment:
@SimonBiggs so I guess the first thing I discovered is that the SQL server instance should only be set up to run for the test runners that run in containers (is this only the ubuntu test job)?

Reply:
This particular job here is Windows, Linux and MacOS. Is there a way to make the SQLServer service start only if the `matrix.task` = 'mosaiq'?